python/ml4ir/applications/classification/model/metrics/metrics_factory.py

```diff
@@ -21,9 +18,7 @@ def get_metric(metric_key: str) -> Metric:
     Metric class
         Class defining the metric computation logic
     """
-    if metric_key == MetricKey.CATEGORICAL_ACCURACY:
```

Comment:
Why was this removed?

Reply:
Never mind. I see it below

Reply:
I removed unnecessary boilerplate. You can simply use `categorical_accuracy` and use tensorflow/keras native loss without overriding and wrapping.